developersociety/commonslibrary
# Add chatacters count for abstract textarea

# Sources
https://trello.com/c/80zAUtS3/9-display-83-of-140-characters-count-down-while-typing-abstract

# Description


# Setup instructions
`fab get_backup`

# How to test
1. Go to http://127.0.0.1:8000/resources/add/ and try to type in the abstract area and it should start showing how many characters you entered
